solutions/cf-20C.mdx

```diff
@@ -76,4 +76,57 @@ int main() {
 
 </CPPSection>
 
+<PySection>
+	
+```py
+
+import heapq
+import sys
+
+
+def dijkstra(graph, source):
+    vis = [False] * (len(graph) + 1)  # initializing each node as unvisited
+    dist = [sys.maxsize] * (len(graph) + 1)  # initializing distance of source to all vertices as +infinity
+    par = [-1] * (len(graph) + 1)  # initializing parent of each node as -1
```

Comment:
remove the comments for these
remember, comments shouldn't explain _what_ you're doing- they should explain _why_ you're doing what you're doing

Reply:
Thank you for the valuable advice, I've made the changes